src/Control/SAMLController.php

```diff
@@ -204,6 +205,14 @@ public function acs()
             $member->$field = $attributes[$claim][0];
         }
 
+        $mapUserGroup = Config::inst()->get(SAMLConfiguration::class, 'map_user_group');
+        // Map user groups
+        if ($mapUserGroup) {
+            $mapper = SAMLUserGroupMapper::singleton();
+
+            $member = $mapper->map($attributes, $member);
```

Comment:
do you need to overwrite the member var? maybe this could happen after the member->write(); below as that's the same point LDAP would kick in?

Reply:
Yes, I came across an issue when trying to add the group and Member hasn't been created yet 😅